gadget/gadget.go

```diff
@@ -715,7 +720,7 @@ func validateRole(vs *VolumeStructure, vol *Volume) error {
 }
 
 func validateBareContent(vc *VolumeContent) error {
-	if vc.UnresolvedSource != "" || vc.Target != "" {
+	if vc.ResolvedSource() != "" || vc.Target != "" {
```

Comment:
this is called while loading the yaml itself afaict, so it needs to stay UnresolvedSource unless